files/setup-05-event-processor.sh

```diff
@@ -22,7 +22,52 @@ ESCAPED_VCENTER_USERNAME=$(echo -n ${VCENTER_USERNAME} | python -c 'import sys,j
 ESCAPED_VCENTER_PASSWORD=$(echo -n ${VCENTER_PASSWORD} | python -c 'import sys,json;data=sys.stdin.read(); print json.dumps(data)[1:-1]')
 ESCAPED_ROOT_PASSWORD=$(echo -n ${ROOT_PASSWORD} | python -c 'import sys,json;data=sys.stdin.read(); print json.dumps(data)[1:-1]')
 
-if [ "${EVENT_PROCESSOR_TYPE}" == "AWS EventBridge" ]; then
+if [ "${EVENT_PROCESSOR_TYPE}" == "Knative" ]; then
+    if [ -z ${KNATIVE_HOST} ]; then
```

Comment:
Engineering nit 🤓  : this assumes the user does not delete the OVF provided `SCHEME`, because otherwise the deployment would silently fail but continue to run. If you don't mind, please also add a check for the `SCHEME`.

Reply:
Not sure what you mean by delete OVF provided scheme? This isn't an option since OVF ensures there's always input ... scheme will always be set to something (Default HTTP) or HTTPS if use toggles it.

Reply:
Ah, it's a toggle and not a string, correct?

Reply:
Correct, so we're good here